src/fileInput/gridReadMatPower.cpp

```diff
@@ -183,8 +184,8 @@ QMIN 5 minimum reactive power output (MVAr)
 VG 6 voltage magnitude setpoint (pu)
 MBASE 7 total MVA base of machine, defaults to baseMVA
 GEN STATUS 8 machine status,
-> 0 = machine in-service
```

Comment:
I fixed some of these comments per http://www.pserc.cornell.edu/matpower/docs/ref/matpower6.0/idx_gen.html but some of them are still strangely corrupted